### ISSUE TITLE
Very large default maximum limit for mu_star

### ISSUE THREAD

**Issue by pat-schmitt** [OGGM/oggm#1410]
I just saw that the default maximum limit for mu_star is 10000 (see https://github.com/OGGM/oggm/blob/master/oggm/params.cfg#L219), but from the explanation above I think it should be rather 1000. Or is there a reason for this large upper limit?

**Comment by pat-schmitt** [comment]
Further, this should raise an error in `mu_star_calibration_from_geodetic_mb` if no limit is given (https://github.com/OGGM/oggm/blob/master/oggm/core/climate.py#L1356)

**Comment by fmaussion** [comment]
Yeah the thing is that very large mus are needed when you don't allow for a temperature bias (cc @lilianschuster). Maybe something to consider for 1.6

**Comment by pat-schmitt** [comment]
Ok thanks.
